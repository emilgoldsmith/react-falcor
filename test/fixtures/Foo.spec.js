import React from 'react'
import renderer from 'react-test-renderer'

import Foo from './Foo'

describe('Foo', () => {
  it('render greeting', () => {
    const component = renderer.create(<Foo greeting="hi"/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
