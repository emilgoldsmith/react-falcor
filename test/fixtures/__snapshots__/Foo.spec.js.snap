exports[`Foo render greeting 1`] = `
<div>
  hi
</div>
`;
